ts/context.ts

```diff
@@ -738,6 +738,9 @@ export class QuickJSContext implements LowLevelJavascriptVm<QuickJSHandle>, Disp
       return this.getNumber(handle)
     } else if (type === "undefined") {
       return undefined
+    } else if (type === "symbol") {
+      const desc = this.getString(this.getProp(handle, "description"))
```

Comment:
I think the property handle is leaked. It needs to be disposed.

Reply:
Fix by this? 
```
      const propHd = this.getProp(handle, "description")
      const desc = this.getString(propHd)
      propHd.dispose()
      return Symbol(desc)
```